internal/pkg/agent/cmd/enroll_cmd.go

```diff
@@ -267,17 +268,23 @@ func (c *enrollCmd) Execute(ctx context.Context, streams *cli.IOStreams) error {
 	}
 
 	defer func() {
-		fmt.Fprintln(streams.Out, "Successfully enrolled the Elastic Agent.")
+		if err != nil {
+			fmt.Fprintf(streams.Out, "Something went wrong while enrolling the Elastic Agent: %v\n", err)
```

Comment:
Should this error be printed to `streams.Err`?
```suggestion
			fmt.Fprintf(streams.Err, "Something went wrong while enrolling the Elastic Agent: %v\n", err)
```

Reply:
I did not see `streams.Err` being used here, so I just followed the pattern. But I agree with you